.github/workflows/automerge.yml

```diff
@@ -13,7 +13,7 @@ jobs:
         uses: actions/checkout@v2
       - name: automerge-action
         id: automerge-action
-        uses: conda-forge/automerge-action@main
+        uses: conda-forge/automerge-action@master
```

Comment:
@conda-forge/conda-smithy, is it an expected change? I do not see the `master` branch at https://github.com/conda-forge/automerge-action/branches, only the `main` one.